src/Entity/PantheonDocument.php

```diff
@@ -110,10 +114,9 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type): a
       ])
       ->setDisplayConfigurable('view', TRUE);
 
-    $fields['status'] = BaseFieldDefinition::create('boolean')
-      ->setLabel(t('Status'))
-      ->setDefaultValue(TRUE)
-      ->setSetting('on_label', 'Enabled')
+    $fields['status']
+      ->setDescription(t('A boolean value indicating whether the document is published. Controlled externally by Content Publisher.'))
+      ->setDisplayConfigurable('form', FALSE)
```

Comment:
This disallows configuring this field on the form in Drupal since we want status controlled from content publisher.